test/acceptance/step_definitions/steps.js

```diff
@@ -0,0 +1,28 @@
+const { Given, When, Then, setDefaultTimeout } = require('@cucumber/cucumber')
+const __ = require('hamjest')
```

Comment:
What's the value of using `hamjest` over the usual Jest matchers?  `hamjest` includes `loadash` as a dependency so is a bit of a beast and we're not really checking for anything that Jest couldn't do without this.